PPTT-2844 Renamed various names of tracking client to "JavaScript Tracking Client" (#663)
PiwikPRO/PPMS-PublicDocs: glossary.rst

.. glossary::

    Analytics

>         PPAS component gathering statistics about each :term:`user` of the :term:`application` (previously **Piwik**).

Reviewer: ```suggestion
        PPAS component gathering statistics about each :term:`visitor` of the :term:`application` (previously **Piwik**).
```

Author: changed

    Analytics attribute
        :term:`Attribute` generated from value provided by :term:`Analytics` (e.g. browser and device data, location
        data, etc.). You can read more about :term:`attribute` sources
        `here <https://help.piwik.pro/audience-manager/data-sources/>`_.

        .. note::
            If :term:`custom attribute` uses the same name - it will be represented as a separate :term:`attribute`.

    Analytics ID

>         ID assigned to :term:`user` by :term:`Analytics` for the duration of :term:`Analytics` session. It is stored in

Reviewer: ```suggestion
        ID assigned to :term:`visitor` by :term:`Analytics` for the duration of :term:`Analytics` session. It is stored in
```

Author: changed

        .. note::
            It is still necessary to identify the :term:`user` with his :term:`analytics ID` to access this information.

    Audience

>         Named set of :term:`attribute` conditions used to define a group of :term:`users<user>` matching them.

Reviewer: ```suggestion
        Named set of :term:`attribute` conditions used to define a group of :term:`visitors<visitor>` matching them.
```

Author: changed

        .. note::
            If :term:`analytics attribute` uses the same name - it will be represented as a separate :term:`attribute`.

    Device ID
        Device ID (device identification) is a distinctive number associated with a smartphone or similar handheld
        device. Device IDs are separate from hardware serial numbers.

    Identifier
        Unique identifier of a :term:`visitor` ID (e.g. :term:`analytics ID`, :term:`user ID`, :term:`device ID` or email).

    JavaScript Tracking Client (JSTC)
        A JavaScript object that is able to send requests to CPP. It is loaded and created with download of `ppms.js` file. It has an :ref:`API<data-collection-javascript-tracking-client-api>` that allows to configure what data requests should contain. You can :ref:`learn more about JSTC here<data-collection-javascript-tracking-client-installation>`

>     JavaScript Tracking Snippet (JSTS)

Reviewer 2: I'm not sure this is a term. Term should be used in multiple places and referenced constantly without explaining what it is (that's where link to this description comes in).

Author: I'm just following orders

        A JavaScript code, usually in form of HTML tag, that initiates JSTC and sends first tracking request. You can see an :ref:`example of JSTS here<jtc-installation-jsts-example>`.

    PII
        Personally Identifiable Information.

>     Tracking Tag

Reviewer 2: I'm not sure this is a term. Term should be used in multiple places and referenced constantly without explaining what it is (that's where link to this description comes in).

Author: I did not add it, cleaning up the glossary can be another task. If you feel so, please fill one.

        .. note::
            If :term:`analytics attribute` uses the same name - it will be represented as a separate :term:`attribute`.

    Device ID
        Device ID (device identification) is a distinctive number associated with a smartphone or similar handheld
        device. Device IDs are separate from hardware serial numbers.

    Identifier

>         Unique :term:`user` ID (e.g. :term:`analytics ID`, :term:`user ID`, :term:`device ID` or email).

Reviewer: ```suggestion
        Unique identifier of a :term:`visitor` ID (e.g. :term:`analytics ID`, :term:`user ID`, :term:`device ID` or email).
```

Author: changed

    JavaScript Tracking Client (JSTC)

>         A JavaScript object that is able to send requests to CPP. It is loaded and created with download of `ppms.js` file. It has an :ref:`API<data-collection-javascript-tracking-client-api>` that allows to configure what data requests should contain. You can :ref:`learn more about JSTC here<data-collection-javascript-tracking-client-installation>`

Reviewer: Please add also the term `CPP`.

Reviewer: Now I see it is introduced in another PR.

    JavaScript Tracking Snippet (JSTS)
        A JavaScript code, usually in form of HTML tag, that initiates JSTC and sends first tracking request. You can see an :ref:`example of JSTS here<jtc-installation-jsts-example>`.

    PII
        Personally Identifiable Information.

    Tracking Tag
        A HTML tag, that is created and configured by Tag Manager. It is loaded to the website with Tag Manager Container. Using a Tracking Tag is an alternative for implementing a JavaScript Tracking Snippet. You can :ref:`learn more about Tracking Tag here<https://help.piwik.pro/support/tag-manager/piwik-pro-tag/>`.

>     User

Reviewer: ```suggestion
    Visitor
```

        Visitor on tracked :term:`application`.

    User ID

>         Permanent ID assigned to :term:`user` by :term:`application` (e.g. username). You can read more about it

Reviewer: ```suggestion
        Permanent ID assigned to :term:`visitor` by :term:`application` (e.g. username). You can read more about it
```

Author: change

        `here <https://help.piwik.pro/tag-manager/userid/>`_.

    Visit

>         Period of continuous :term:`user` activity on :term:`application`. It ends in the following situations:

Reviewer: ```suggestion
        Period of continuous :term:`visitor` activity on :term:`application`. It ends in the following situations:
```

Author: changed

            - after a period of inactivity (option set to 30 minutes by default)

>             - at midnight (option enabled by default)

Reviewer: This is not true any more. Sessions are not split if midnight occurs in new tracker (new tracker is not aware of tracked app's time zone).

Author: Removed
